tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -96,7 +102,7 @@ Status RetryingFileSystem::NewRandomAccessFile(
   TF_RETURN_IF_ERROR(CallWithRetries(std::bind(&FileSystem::NewRandomAccessFile,
                                                base_file_system_.get(),
                                                filename, &base_file)));
-  result->reset(new RetryingRandomAccessFile(std::move(base_file)));
+  result->reset(new RetryingRandomAccessFile(std::move(base_file), 0));
```

Comment:
It looks like if you pass zero for the initial_delay, it will effectively disable exponential backoff. Did you want to pass initial_delay_seconds_ which is a member of RetryingFileSystem?